### ISSUE TITLE
jep.jarray with only one arg causes segfault

### ISSUE THREAD

**Issue by ttgurney** [mrj0/jep#59]
```
$ jep
>>> jep.VERSION
'3.7.0'
>>> jep.jarray(0)
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f213d88e45d, pid=25541, tid=0x00007f21593cd700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_101-b13) (build 1.8.0_101-b13)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.101-b13 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [jep.so+0xc45d]  pyjarray_new_v+0xdd
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/tgurney/jep-git/jep/hs_err_pid25541.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Aborted (core dumped)

```

**Comment by ttgurney** [comment]
Looks like this bug has existed since the beginning of time. Not sure whether the one-arg form is supposed to be allowed or not.

**Comment by ndjensen** [comment]
Fixed in 3.7
